.prettierignore

```diff
@@ -0,0 +1,60 @@
+# No need to run prettier here
+*.md 
+*.yml
+*.html
+*.json
+
+
+# Compiled output
+/packages/core/**/*.js
+/packages/core/**/*.d.ts
+/packages/shared-internals/**/*.js
+/packages/shared-internals/**/*.d.ts
+/packages/compat/**/*.js
+/packages/compat/**/*.d.ts
+/packages/macros/**/*.js
+/packages/macros/**/*.d.ts
+/packages/util/src/**/*.js
+/packages/util/src/**/*.d.ts
+/packages/webpack/**/*.js
+/packages/webpack/**/*.d.ts
+/packages/hbs-loader/**/*.js
+/packages/hbs-loader/**/*.d.ts
+/test-packages/support/**/*.js
+/test-packages/**/*.d.ts
+/tests/scenarios/**/*.js
+/tests/scenarios/**/*.d.ts
+/packages/test-setup/**/*.js
+/packages/test-setup/**/*.d.ts
+/packages/addon-shim/**/*.js
+/packages/addon-shim/**/*.d.ts
+/packages/addon-dev/**/*.js
+/packages/addon-dev/**/*.d.ts
+/tests/fixtures/
+
+# unconventional js
+/blueprints/*/files/
+/vendor/
+
+# compiled output
+/packages/*/dist/
+/packages/*/tmp/
+/test-packages/*/dist/
+/test-packages/*/tmp/
+
+# dependencies
+bower_components/
+node_modules/
+
+# misc
+coverage/
+!.*
```

Comment:
I think this is in the default bp but does it actually make sense? isn't this just re-adding all files?

Reply:
it should only be re-adding `.`-prefixed files, like `.eslintrc.js`